seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oceanbase/OceanBaseMysqlType.java

```diff
@@ -0,0 +1,567 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.oceanbase;
+
+import com.mysql.cj.util.StringUtils;
```

Comment:
Have you tested creating table without mysql driver? IMO this util shoud not be introduced here.

Reply:
I'm sorry for the error caused by the code rollback, I've modified it and retested it，There is no MySQL driver in this lib, only Oceanbase, which can automatically create tables
![image](https://github.com/user-attachments/assets/ae24e69a-4961-494a-8a13-c9b95328ace4)
![image](https://github.com/user-attachments/assets/f8684436-3033-44ec-901f-02a7a3b045f3)